java/code/src/com/suse/manager/webui/utils/salt/custom/UptimeTrackerRequestData.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2024 SUSE LLC
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+
+package com.suse.manager.webui.utils.salt.custom;
+
+import com.suse.salt.netapi.results.CmdResult;
+import com.suse.salt.netapi.results.StateApplyResult;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.Optional;
+
+/**
+ * Suse Uptime Tarcker Result Data from uptimetracker.requestdata
+ */
+public class UptimeTrackerRequestData {
+
+    @SerializedName("cmd_|-dump_uptime_info_|-cat /etc/zypp/suse-uptime.log|" +
```

Comment:
This is more about the tool which produce the data than this code here, but is "/etc" a good directory for having something which is called ".log" ?
"/etc" is for configuration files. I think "/var" would be better for this. But this is up to you.

Reply:
I am not sure why /etc/zypp/ was selected. The only reason I can think of would be that the filesystem is guaranteed to be present and local to the system, but I have no control of the location.